MIGRATION.md

```diff
@@ -126,7 +126,7 @@ The following components do not have a codemod, so you must migrate them manuall
 
 - [Accordion](#accordion)
 - [Asset](#asset)
-- [Autocomplete](v4.f36.contentful.com/components/autocomplete/)
+- [Autocomplete](https://v4.f36.contentful.com/components/autocomplete/)
```

Comment:
Can you change it pls to `https://f36.contentful.com/components/autocomplete` . No need for `v4` anymore